frontend/conf/PROD.public.conf

```diff
@@ -52,3 +52,6 @@ google.adwords.joiner.conversion {
 }
 
 logstash.enabled=false
+
+eventbrite.api.refresh-time-seconds=299
+eventbrite.api.refresh-time-priority-events-seconds=119
```

Comment:
reduced these to try to prevent rate limit errors.  We can always bump further or reduce the size of the ASG to help further.